backend/algorithms/errors.json

```diff
@@ -7211,127 +7578,121 @@
         ],
         "broke at": "Index 6, /CODE"
     },
-    "COMP3901": {
+    "COMM1120": {
```

Comment:
im not sure how this passed our test, but i think this should be a manual fix. Sorry that i have to put this on you to do lol

Reply:
will do, also looks like the processing stage for COMM3091 is buggy. I think we're normally able to process conditions like that